### PR TITLE
Use direct `test.config.js` existence check.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
  * Copyright (c) 2016-2022 Digital Bazaar, Inc. All rights reserved.
  */
 import * as bedrock from '@bedrock/core';
+import {existsSync} from 'node:fs';
```
Comment (dlongley) [inline lib/index.js:5]: Sync? So sad... :(

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # bedrock-test ChangeLog
 
+## 8.1.0 - 2022-xx-xx
```
Comment (dlongley) [inline CHANGELOG.md]: Why not `8.0.4` if this is just a fix?

**Review by dlongley — APPROVED** [review]

File: lib/index.js (hunk 1 above)
Comment (davidlehn) [reply lib/index.js:5]: It matters little in this case.  But if you can decipher the docs related to async exists with stat/access... please provide a patch!

File: CHANGELOG.md (hunk 2 above)
Comment (davidlehn) [reply CHANGELOG.md]: I can change that.  The patch I started with was complex enough to seem like more than a simple fix.  It's better now.

**Pull request merged**